korman/ui/modifiers/physics.py

```diff
@@ -20,6 +20,7 @@ def collision(modifier, layout, context):
 
     split = layout.split()
     col = split.column()
+    col.prop(modifier, "dynamic_blocker")
```

Comment:
Can you move this down to below `camera_blocker`? My CDO insists that everything be alphabetized.